src/core/Button/Button.baseStyles.tsx

```diff
@@ -121,7 +113,11 @@ export const baseStyles = withSuomifiTheme(
     user-select: none;
   }
 
-  ${fullWidth && fullWidthStyles}
+  &.fi-button--fullWidth {
```

Comment:
Use lowercase in CSS-classes - with single dash or without

Reply:
Fixed.